scripts/README.md

```diff
@@ -0,0 +1,176 @@
+# Deployment scripts
+
+This directory contains two types of script: [operate test and discovery
+environments](../create-a-discovery-register-environment.md) and upload data
+to the Open Register Java (ORJ) application.
+
+
+## Introduction
+
+The raw data for every GDS-hosted register is in Github:
+https://github.com/openregister/registry-data
+
+The data is stored as Register Serialisation Format (RSF), a plain text
+line-based format that allows encoding data and metadata in a single file.
+
+ORJ also expects a patch encoded as RSF when receiving an update request.
+
+Previous versions used YAML and TSV in a repository-per-register approach with
+the naming convention `{register-identifier}-data`. This is now deprecated in
+favour of a single repository `registry-data`.
+
+
+## Phases (environments)
+
+A register can be deployed in different environments, currently the following
+exist:
+
+* `test`: For throwaway tests. E.g. `https://country.test.openregister.org`.
+* `discovery`: Private feedback phase. E.g. `https://country.cloudapps.digital`.
+* `alpha`: Public feedback phase. E.g. `https://country.alpha.openregister.org`.
+* `beta`: Live phase. E.g. `https://country.beta.openregister.org`.
+
+
+## Setup
+
+### Dependencies
+
+Before attempting to use any update operation, please ensure all dependencies
+are correctly installed and set up in your machine.
+
+* [Registers Credentials set up](https://github.com/openregister/credentials).
+* Python 3.
```

Comment:
should we specify Python 3.7?

Reply:
Pipenv locks the version to 3.7 already (one of the reasons for using Pipenv).